scrapy/extensions/feedexport.py

```diff
@@ -317,6 +367,18 @@ def _exporter_supported(self, format):
             return True
         logger.error("Unknown feed format: %(format)s", {'format': format})
 
+    def _settings_are_valid(self, uri):
+        """
+        If FEED_STORAGE_BATCH_ITEM_COUNT setting is specified uri has to contain %(batch_time)s or %(batch_id)s
+        to distinguish different files of partial output
+        """
+        if not self.storage_batch_item_count or '%(batch_time)s' in uri or '%(batch_id)s' in uri:
+            return True
+        logger.error(
+            '%(batch_time)s or %(batch_id)s must be in uri if FEED_STORAGE_BATCH_ITEM_COUNT setting is specified'
```

Comment:
This message might need an update to reflect that fact that the batch size could come from `FEEDS.batch_item_count`.